packages/salesforcedx-apex-replay-debugger/package.json

```diff
@@ -45,7 +45,7 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "jest --coverage",
+    "test:unit": "jest --coverage --forceExit",
```

Comment:
On CI this test suite was hanging.  To get past it I added the forceExit option

Reply:
It is until we update the debugger test dev dep that we have.  It has a method that listens for a vscode event and exits early.  Wasn't an issue on the old version of jest, but with the new version it correctly errors.